Add route to retrieve configs filtering by client_id (#149)
omnivector-solutions/license-manager: backend/CHANGELOG.rst

Unreleased
----------
* Add cluster_id column to config table to identify which cluster the configuration applies to

> * Created new route to fetch configurations filtering by the cluster requesting the data

Reviewer: I am not sure what this means. Is the following correct?
```suggestion
* Added new route to fetch all configuration from a specific cluster
```

Author: Yes. This route is going to be used only by the agents. It identifies which cluster is requesting the data and automatically filters the configs.

Author: I changed it in the latest commit to use your suggestion, thanks!
